test/modules/standard/Vector/capacities.chpl

```diff
@@ -1,35 +0,0 @@
-use Containers;
```

Comment:
Do these tests have equivalents for the vector functionality on DefaultRectangular?